alica_tests/src/test/etc/Alica.yaml

```diff
@@ -0,0 +1,120 @@
+#Local:
+## optional id, if missing, engine will generate a random id
+#  ID: 12345678
+#  DefaultRole: DEFAULT_ROLE_TEST
+#  <AGENTCAPABILITY_1>: TEST_CAPABILITY_1
+#  <AGENTCAPABILITY_2>: TEST_CAPABILITY_1
+
+Local:
+  nase:
+    ID: 9
+    DefaultRole: Defender
+    AverageTranslation: 3000.0
+    MaxTranslation: 3000.0
+    AverageRotation: 3.1415
+    IsOmnidrive: true
+    Kicker: True
+    IsGoalie: False
+    Speed: Medium
+    CanPass: True
+    LocalizationSuccess: -1
+    HasActiveBallHandler: true
+
+  hairy:
```

Comment:
In tests have different yaml files for nase and hairy , pass the appropriate path while creating respective AlicaContext